README.md

```diff
@@ -198,6 +203,49 @@ To enable this on iOS simulators you need to do the following (when you have a s
 
 In the previous mentioned menu you can also select a (non)matching Touch / Face ID when the phone is asking for it.
 
+## Deep linking
+This app supports deep linking for iOS and for Android, this means that screens can directly be opened with a deep link.
+
+The prefix deep link is `swaglabs://` and the following screens (with their arguments) can be used:
+
+- **Swag overview screen:** `swag-overview/ids` where `ids` is a string of numbers from 0-5 separated with a `,`. For example `swag-overview/0,2`. The number represents a product. 
```

Comment:
It might be good to clarify in this doc that specifying these IDs will add the items to the cart - it appears that the inventory list and checkout pages add items by ID to the shopping cart. As a reader, I would have expected the page to display those items, not necessarily add them to the cart.

Also good to clarify that these IDs are optional (at least they appear to be based on my read of the code)

Reply:
Done and thanks